src/app/services/user-service/user.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { User } from 'src/app/models/user';
+import { Batch } from 'src/app/models/batch';
+
+@Injectable({
+  	providedIn: 'root'
+})
+export class UserService {
+
+	url: string = 'http://localhost:8080/users';
```

Comment:
Let's start using the environment files to parameterize these value; this wont always be "localhost"